src/main/java/seedu/trackbeau/model/customer/HairType.java

```diff
@@ -0,0 +1,57 @@
+package seedu.trackbeau.model.customer;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.trackbeau.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's hair type in the trackbeau book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidHairType(String)}
+ */
+public class HairType {
+
+    public static final String MESSAGE_CONSTRAINTS = "HairTypes can be any values and it should not be blank";
+
+    /*
+     * The first character of the trackbeau must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
+
+    public final String value;
+
+    /**
+     * Constructs an {@code hairType}.
+     *
+     * @param hairType A valid hair type.
+     */
+    public HairType(String hairType) {
```

Comment:
Are we going to change the user guide for the skin and hair type? Because in the UG it states that multiple skin and hair types can be added.

Reply:
edited

Reply:
Sorry, please ignore what I mentioned as I misread the UG.